src/index.css

```diff
@@ -11,3 +11,158 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+input{
+  padding:0;
+  margin:0;
+}
+label {
+  padding: 0;
+  margin: 0;
+}
+
+.wrapper{
+  max-width: 1280px;
+  width: 90%;
+  margin: 0 auto;
+}
+html{
+  font-size: 125%;
+}
+ul{
+  list-style: none;
+  padding:0;
+  margin:0;
+  margin-bottom: 25px;
+}
+
+header{
+  background-image:url("../public/simpleTravelBanner.jpg");
+  background-position: center;
+  height:80vh;
+}
+header .wrapper{
+  padding-right: 50px;
+  display:flex;
+  flex-direction: column;
+  align-items: flex-end;
+  justify-content: center;
+  height:100%;
+}
+h1{
+  color:rgba(31, 22, 22); 
+  font-size: 3.25rem;
+  text-shadow: 3px 2px black;
+}
+.cityInfo{
+  display: flex;
+  align-items: center;
+  padding: 10px 0;
+}
+h3{
+  font-size: 1.75rem;
+  margin:0;
+  margin-right: 25px;
+}
+
+.restaurantList{
+  border-bottom: 2px solid black;
+  display:flex;
+  margin-bottom:10px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.info{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.info div{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 75%;
+}
+
+.info p{
+  margin-right: 25px;
+  font-size: 1rem;
+  text-transform: capitalize;
+  margin-left: 25px;
+}
+form{
+  display:flex;
+  align-items: center;
+  margin:25px 0 30px 0;
+  padding:0;
+}
+form label{
+  font-size: 2rem;
+}
+
+input{
+  margin: 0 15px;
+  height:18px;
+}
+.display{
+  margin:0;
+  padding:0;
+}
+
+.display p, .display a{
+  text-align: center;
+
+}
+
+footer{
+  background-color:rgba(31, 22, 22);
+  color:white;
+}
+footer p{
+  padding: 25px 0;
+  margin:0 auto;
+  text-align: center;
+}
+footer p:first-child{
+  padding-bottom:0;
+}
+
+@media(max-width: 800px){
+  header .wrapper{
+    padding-right: 0;
+    text-align: center;
+  }
+  /* h1, h2{
+    text-align: center;
+  } */
+}
+
+@media(max-width: 600px){
+  .info{
+    flex-direction: column;
+    align-items: flex-start;
+    margin-bottom: 15px;
+  }
+
+  .display{
+    display: flex;
```

Comment:
maybe do a flex start to align with the info section and potentially remove left/right margins on p?